tasks/create.yaml

```diff
@@ -2,6 +2,9 @@ variables:
   - name: FLAVOR
     default: upstream
 
+  - name: REGISTRY1_PEPR_IMAGE
+    default: registry1.dso.mil/ironbank/opensource/defenseunicorns/pepr/controller:v0.27.0
```

Comment:
Will want to make sure this is tracked by renovate - not sure the current config would pick this up?

Reply:
Yeah i was hoping renovate would pick it up, but i think you're right that it won't. changing one of the current docker regex matchers to work with the runner vars format since it doesn't look to be used anywhere else.